sigs.yaml

```diff
@@ -90,6 +166,63 @@ sigs:
         description: Design Proposals
       - name: sig-apps-test-failures
         description: Test Failures and Triage
+    subprojects:
+    - name: chart-testing
+      owners:
+        - https://github.com/kubernetes-helm/chart-testing/blob/master/OWNERS
+    - name: chartmuseum
+      owners:
+        - https://github.com/kubernetes-helm/chartmuseum/blob/master/OWNERS
+    - name: charts
+      meetings:
```

Comment:
Does this mean meetings are in two different places?

Reply:
the idea is that subprojects may have their own meetings independent of the entire sig

Reply:
Which makes sense, but for example, the helm/charts calls are already defined elsewhere in sigs.yaml (https://github.com/kubernetes/community/blob/02d7d8e61aaf888a583b813054681f61424f8b06/sigs.yaml#L63-L85), and the templates know how to look for those. Perhaps there just needs to be a way to associate subprojects with the meetings?

Biggest objection is to defining meeting information in two places

Reply:
Oops, I meant to move those meetings down into the subprojects, and forgot (I assumed there'd be a merge conflict on rebase)... but it does mean you don't see all meetings for a SIG and all of its subprojects at a glance in sigs-list.md

I'm open to changing one way or another but I'm going to default to DRY meetings defined only for the relevant subproject or SIG and await further feedback